build/build.yml

```diff
@@ -8,9 +8,9 @@ pool:
 
 steps:
   - task: NodeTool@0
-    displayName: "Use Node 14.x"
+    displayName: "Use Node 16.x"
```

Comment:
the @types/node version in the package.json is 18.11.9. Should the pipeline be updated to node 18 instead of 16?

Reply:
Sure, let's test :)